.pre-commit-config.yaml

```diff
@@ -15,6 +15,13 @@ repos:
         stages: [commit]
         log_file: .black.log
 
+      - id: blacken-docs
```

Comment:
FR to support `test-code-block`: https://github.com/adamchainz/blacken-docs/issues/214